### ISSUE TITLE
Enable lint rules with Kotlin

### ISSUE THREAD

**Issue by hotchemi** [permissions-dispatcher/PermissionsDispatcher#373]
Even after migrating based API to UAST, currently our lint rules only work for Java.
It seems Kotlin's support is still not available in 3.0 and Android tools team is [planning to include this enhancement in 3.1](https://groups.google.com/d/msg/lint-dev/2kLx2-k4yJU/i9mio2OSAQAJ).
So currently we gotta wait it being came out 🤔

**Comment by aurae** [comment]
What a bummer. Let's wait it out then! The groundworks has already been done, which was the most challenging part.

**Comment by Naitbit** [comment]
It seems to run now with Kotlin. I got (erroneous) lint report for my Kotlin code with
```
buildToolsVersion '26.0.1'
compile 'com.github.hotchemi:permissionsdispatcher:2.4.0'
kapt 'com.github.hotchemi:permissionsdispatcher-processor:2.4.0'

gradle wrapper 4.1 
kotlin_version '1.1.51' 
com.android.tools.build:gradle  '2.3.3'
```

**Comment by shiraji** [comment]
This should be closed by Android Studio v3.1's release, right?

**Comment by hotchemi** [comment]
Thx for heads up! According to [release note](https://developer.android.com/studio/releases/index.html#3-1-0), `Kotlin code now analyzed with command-line lint check`🎉
We haven't tested every lint works well Koltin so then next step is adding more test cases which cover Kotlin, am I correct? 👀
